core/src/main/scala/kafka/server/BrokerServer.scala

```diff
@@ -340,7 +338,8 @@ class BrokerServer(
         SocketServer.DataPlaneThreadPrefix)
 
       if (socketServer.controlPlaneRequestChannelOpt.isDefined) {
-        throw new RuntimeException("KIP-291 endpoints are not supported when in KRaft mode.")
+        throw new RuntimeException("KIP-291 control plane listeners are not supported " +
```

Comment:
nit: you might have missed my comment. Can we mention the configuration `control.plane.listener.name` explicitly instead of KIP-291?

Reply:
ok